### PR TITLE
Simulator event rendering fixes

### PR DESCRIPTION
**Pull request by rowanseymour**
* Render input_labels_added events (#790)
* Fix rendering of contact_groups_changed events (#804)
* Adds tests!

### CODE REVIEW

File: src/config/i18n/defaults.json [hunk 1]
```diff
@@ -300,12 +300,13 @@
...
     "flow_entered": "Entered flow \"[[flow]]\"",
     "httplog_dialog": "HTTP Request Details",
+    "input_labels_added": "Message labeled with ",
     "msg_wait": "Waiting for reply",
     "removed_from_group": "Removed from ",
     "resthook_called": "Triggered flow event \"[[resthook]]\"",
     "run_result_changed": "Set result \"[[field]]\" to \"[[value]]\"",
     "sent_email": {
-      "summary": "Sent email to \"[[recipients]]\" with subject \"[[subject]]\"",
+      "summary": "Sent email to [[recipients]] with subject \"[[subject]]\"",
```
Comment (rowanseymour) [inline src/config/i18n/defaults.json]: Fixes whole list having quotes instead of each item in the list

File: src/components/simulator/__snapshots__/LogEvent.test.tsx.snap [hunk 2]
```diff
@@ -0,0 +1,324 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (rowanseymour) [inline src/components/simulator/__snapshots__/LogEvent.test.tsx.snap]: big fan of snapshot testing now - it's the only sensible way to check HTML outputs

Comment (ericnewcomer) [reply src/components/simulator/__snapshots__/LogEvent.test.tsx.snap]: Totally agree. Also, while probably not a huge deal for the log event stuff, but as a general rule for snapshot testing, I try to have pertinent asserts along with the snapshot. Otherwise, it can be really easy to update a snapshot without realizing you are changing an expected truth.

Here is an example:
https://github.com/nyaruka/floweditor/pull/801/files#diff-05f981483ae9b76ba83005bfafd192b3R45

Basically, just treat the snapshot as a bonus assertion.
